ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -46,18 +46,22 @@ export class InstallationProgress extends React.Component {
                         "org.fedoraproject.Anaconda.Task",
                         tasks[0]
                     );
+
                     const addEventListeners = () => {
                         taskProxy.addEventListener("ProgressChanged", (_, step, message) => {
+                            if (step === 0) {
+                                getSteps({ address: this.context, task: tasks[0] })
+                                        .then(
+                                            ret => this.setState({ steps: ret[0].v }),
```

Comment:
So I get that `ret` is the data returned by async call, and its apparently some array, so `ret[0]` gives you the first item from that array. But where does the `ret[0].v` come from ? :-D

Reply:
The anaconda dbus API returns properties in a variant object. So it's an object with keys, 'v' for value and 't' for type.

Reply:
did the [0] part in the API method wrapper, to remove the complexity of the react side.

Reply:
Oh, interesting - I was thinking its either value or variant, that explains it. :)